Overview.src.html

```diff
@@ -3358,6 +3365,27 @@ <h4 id="should-response-to-request-be-blocked-due-to-nosniff?"><dfn title="shoul
  <li><p>Return failure.
 </ol>
 
+<h3 id="should-response-to-request-be-blocked-due-to-mime-type?"><dfn title="should response to request be blocked due to mime type">Should
+<var>response</var> to <var>request</var> be blocked due to its MIME type?</dfn></h3>
+
+<p>Run these steps:
+
+<ol>
+ <li><p>Let <var>MIMEType</var> be the result of
+ <span title=concept-header-extract-mime-type>extracting a MIME type</span> from
+ <var>response</var>'s <span title=concept-response-header-list>header list</span>.
+
+ <li><p>Let <var>type</var> be <var>request</var>'s
+ <span title=concept-request-type>type</span>.
+
+ <li><p>If <var>type</var> is "<code title>script</code>", and <var>MIMEType</var>
+ begins with `<code title>image/</code>`, return <b title>blocked</b>.
+
+ <li><p class=XXX>"<code title>audio</code>", "<code title>video</code>" ...
```

Comment:
Is this actually something we want to do? Maybe just leave this out until we actually get there?